src/librustc_mir/borrow_check/nll/type_check/liveness/trace.rs

```diff
@@ -0,0 +1,544 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
```

Comment:
can you summarize either what this module is doing, or add a doc comment (with similar purpose) to the `fn trace` function below?

Reply:
(the word "trace" has various connotations for me and I figure its better to just explicit say what you are getting at at the outset of either the module or the function...)

Reply:
I guess the comments I am looking for can be found above `fn compute_use_live_points_for` and `fn compute_drop_live_points_for` ...